c2corg_ui/templates/document/diff.html

```diff
@@ -20,7 +20,7 @@ <h2 class="text-center">${title}</h2><br>
         <a href="${request.route_url(
               module + '_archive', id=document_id, lang=lang,
               version=version1['version_id'])}"><span translate>Revision as of</span> {{'${1000 * version1['written_at']}' | date:'medium'}}</a><br/>
-        <span translate>by</span> <a href="#TODO profile for ${version1['user_id']}">${version1['name']}</a>
+        <span translate>by</span> <a href="${request.route_url('profiles_view_id', id=version1['user_id'])}">${version1['name']}</a>
```

Comment:
Indeed the user locale is not available when retrieving the archived versions I think. That's why I use the "id only" link.
Having the user locale (but not the whole description etc.) would make sense to create the whole profile link.
But it's not very critical ("nice to have") because the "id only" link also works (but triggers additional requests to get the lang when loaded).